packages/helium-admin-cli/emissions/mobile.json

```diff
@@ -3,12 +3,20 @@
     "startTime": "2023-04-18T00:00:00Z",
     "emissionsPerEpoch": "108,493,150.684932"
   },
+  {
+    "startTime": "2023-07-28T00:00:00Z",
+    "emissionsPerEpoch": "174,246,575.342466"
+  },
+  {
+    "startTime": "2023-07-29T00:00:01Z",
+    "emissionsPerEpoch": "141,369,863.013699"
```

Comment:
Represents increase of 20% mapper rewards

Reply:
confirmed post halving number.